extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -106,7 +120,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    const format = `${slotRect.width}x${slotRect.height}`;
+    const width = slotRect.width;
+    // If responsive, set height to value that is considered valid by
+    // ad server.
+    const height =
+        this.isResponsive_ ? Math.ceil(width / 1.91 + 120) : slotRect.height;
```

Comment:
instead of doing a resize later, seems to me the ideal solution is make it the right size when we first lay things out?

Probably we can introduce an overridable function `calculateHeight` ? @dvoytenko